taskcluster/ci/config.yml

```diff
@@ -13,20 +13,26 @@ taskgraph:
             # string: the name of the repo, for humans
             name: "XPI manifest"
             # regex: to match the base name of the repo
-            project-regex: xpi-manifest
+            project-regex: xpi-manifest$
             # string: the repo url. Use `git@github.com:org/repo` for private
             #         repositories.
             default-repository: https://github.com/mozilla-extensions/xpi-manifest
             # string: which branch is primary; should be `master`
             default-ref: master
             # enum: hg or git; should be `git`
             type: git
-        template:
-            name: "template repo test"
+        stagingpublic:
```

Comment:
We may want to update the `repo-prefix` in `manifests/template-example.yml.template` since that still refers to `template`.